runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -0,0 +1,816 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2021 IBM Corp. and others
```

Comment:
Copyright year needs to be changed, Like wise in the previous `VectorAPIExpansion.cpp` file.

Reply:
I thought this is what we are supposed to specify...

Reply:
I believe it should say "2021, 2021" with the first number indicating the original date of copywrite.

Also, the copywriter year in Optimizations.enum needs to be updated.

Reply:
Fixed!